### DISCUSSION TITLE
Does wezterm supports using mouse to set text cursor position like iterm2?

### DISCUSSION THREAD

**Discussion by amardeep** [wezterm/wezterm#3049] [Q&A]
In iterm, using opt + click changes text cursor position:

From: https://iterm2.com/documentation-preferences-pointer.html

> Option-Click moves cursor
> If enabled, option-click will move the cursor to where the mouse pointer is. If you install shell integration, this will be well-behaved at the shell prompt by not sending up and down arrow keys.

Is this supported by wezterm?

**Comment by wez** [comment]
wezterm does not have this feature

**Reply by amardeep** [reply]
Thanks for the quick response.

Will be nice to have this feature. Comes handy sometimes when trying to edit long command lines.

**Reply by wez** [reply]
Follow along on:
* https://github.com/wez/wezterm/issues/1175
